integration-tests/utils/integration-helpers.js

```diff
@@ -15,7 +15,7 @@ const withSignedInPage = fn => async options => {
   const page = await context.newPage()
   try {
     await signIntoTerra(page, { token, testUrl })
```

Comment:
Could page logging be setup before `signIntoTerra` is called? So that any errors in `signIntoTerra` are logged.

Reply:
Had an idea. Created new `enablePageLogging` func, it is called inside `withPageLogging` function.